tls/s2n_config.h

```diff
@@ -105,3 +110,8 @@ int s2n_config_free_session_ticket_keys(struct s2n_config *config);
 void s2n_wipe_static_configs(void);
 extern struct s2n_cert_chain_and_key *s2n_config_get_single_default_cert(struct s2n_config *config);
 int s2n_config_get_num_default_certs(struct s2n_config *config);
+
+int s2n_config_clear_all_curves(struct s2n_config *conf);
+int s2n_config_add_keyshare_by_group(struct s2n_config *conf, uint16_t iana_id);
+int s2n_config_send_empty_keyshare(struct s2n_config *conf);
+int s2n_config_add_all_curves(struct s2n_config *conf);
```

Comment:
Are we sure this is the interface we want?
* If we aren't, and it's just to test the HRR, then we should put "in test" checks on these.
* If we are, then we need to add these to the api and write documentation.

Reply:
I am thinking of leaving this in test for now, as we require it to test HRR.